README.md

```diff
@@ -91,9 +93,9 @@ There are two ways to work on this project: either by working against an existin
 
 To run the developer version of both the language server and the VSCode extension:
 
-- open the root of this repo in VSCode
-- Go to the debugging panel
-- Make sure "Run VSCode Extension" is selected, and hit run
+-   open the root of this repo in VSCode
```

Comment:
it doesn't seem like we need the extra spaces here. I noticed the same in `svelte-check/README.md`

Reply:
That's prettier formatting it that way.